dev/archery/archery/benchmark/google.py

```diff
@@ -67,8 +67,7 @@ def results(self, repetitions=1):
 class GoogleBenchmarkObservation:
     """ Represents one run of a single (google c++) benchmark.
 
-    Observations are found when running with `--benchmark_repetitions`. Sadly,
```

Comment:
`--benchmark_repetitions` is the Google Benchmark option. It's just `--repetitions` in Archery.

Reply:
Hmm, since this is the doctring for the `GoogleBenchmarkObservation`, I think it's ok to mention the Google Benchmark option.

Reply:
I've updated the docstrings with the text suggested by Ben. It mentions both flags (`--benchmark_repetitions` & `--repetitions`).